public/docs/ts/latest/guide/structural-directives.jade

```diff
@@ -15,20 +15,19 @@ style.
   - [What are structural directives?](#definition)
   - [*NgIf* case study](#ngIf)
   - [Group sibling elements with &lt;ng-container&gt;](#ng-container)
-  - [The asterisk (\*) prefix](#asterisk)
-  - [Inside *NgFor*](#ngfor)
+  - [The asterisk (*) prefix](#asterisk)
+  - [Inside *NgFor*](#ngFor)
     - [microsyntax](#microsyntax)
     - [template input variables](#template-input-variable)
     - [one structural directive per element](#one-per-element)
   - [Inside the *NgSwitch* directives](#ngSwitch)
-  - [Prefer the (\*) prefix](#prefer-asterisk)
+  - [Prefer the (*) prefix](#prefer-asterisk)
   - [The &lt;template> element](#template)
   - [Write a structural directive](#unless)
 
   Try the <live-example></live-example>.
 
-a#definition
-.l-main-section
+.l-main-section#definition
```

Comment:
Please don't "fix" these. I've created separate anchors on purpose, even where the header text would match the anchor name.  I break fewer links this way and don't have to worry about what happens when I change a heading. I can also quickly find something when editing. So these kinds of changes actually make my life harder.

Most importantly, we're converting to pure markdown in v.2 and this makes it harder to figure out what's going on. 

**TAKE AWAY: Please revert them all and do not make further changes to structure/layout that have no visual effect**